`linkerd profile` panic when passing in bad data to `--open-api`

## Bug Report

### What is the issue?

I was creating a service profile but forgot to start my local swagger service. curl threw a `connection refused` and the linkerd cli panicked

### How can it be reproduced?

```bash
curl http://localhost:8080/swagger/v1/swagger.json | l profile -n myns --open-api - myservice | k apply -f -
```

### Logs, error output, etc

```text
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed to connect to localhost port 8080: Connection refused
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x107a4c2]

goroutine 1 [running]:
github.com/linkerd/linkerd2/cli/cmd.renderOpenAPI(0xc4204a1700, 0x147f640, 0xc42000c018, 0x1369b21, 0x7)
        /go/src/github.com/linkerd/linkerd2/cli/cmd/profile.go:167 +0x402
github.com/linkerd/linkerd2/cli/cmd.newCmdProfile.func1(0xc4205fc240, 0xc4200baf00, 0x1, 0x5, 0x0, 0x0)
        /go/src/github.com/linkerd/linkerd2/cli/cmd/profile.go:111 +0x110
github.com/spf13/cobra.(*Command).execute(0xc4205fc240, 0xc4200baeb0, 0x5, 0x5, 0xc4205fc240, 0xc4200baeb0)
        /go/src/github.com/spf13/cobra/command.go:698 +0x46d
github.com/spf13/cobra.(*Command).ExecuteC(0x1bf5fe0, 0x0, 0xc4203b1f68, 0x108ba0f)
        /go/src/github.com/spf13/cobra/command.go:783 +0x2e4
github.com/spf13/cobra.(*Command).Execute(0x1bf5fe0, 0xc4203b1f78, 0xc4200b2058)
        /go/src/github.com/spf13/cobra/command.go:736 +0x2b
main.main()
        /go/src/github.com/linkerd/linkerd2/cli/main.go:10 +0x2d
error: no objects passed to apply
```

### Environment

- Kubernetes Version: 1.13.1
- Cluster Environment: AKS-Engine
- Host OS:
- Linkerd version: Client version: edge-18.12.4

### Possible solution

### Additional context

--- a/cli/cmd/profile.go
+++ b/cli/cmd/profile.go
@@ -164,10 +164,12 @@ func renderOpenAPI(options *profileOptions, w io.Writer) error {
 	routes := make([]*sp.RouteSpec, 0)
 
 	paths := make([]string, 0)
-	for path := range swagger.Paths.Paths {
-		paths = append(paths, path)
+	if swagger.Paths != nil {
+		for path := range swagger.Paths.Paths {
+			paths = append(paths, path)
+		}
+		sort.Strings(paths)
 	}
-	sort.Strings(paths)
 
 	for _, path := range paths {
 		item := swagger.Paths.Paths[path]

